src/PhpSpec/ObjectBehavior.php

```diff
@@ -32,6 +32,9 @@
  * @method void beConstructedThrough($factoryMethod, array $constructorArguments)
  * @method void beAnInstanceOf($class)
  * @method void shouldHaveType($type)
+ * @method void shouldImplement($class)
+ * @method void shouldBeAnInstanceOf($class)
+ * @method void shouldReturnAnInstanceOf($class)
```

Comment:
The methods are fulfilled through a magic method.  Isn't that the use-case for adding @method calls to the class definition?